client/app/dedicated/server/interfaces/interfaces.html

```diff
@@ -0,0 +1,200 @@
+<h1 data-translate="dedicated_server_interfaces_title"></h1>
+<p data-translate="dedicated_server_interfaces_introduction"></p>
+<p>
+    <span data-translate="dedicated_server_interfaces_vrack_information"></span>
+    <a href="https://docs.ovh.com" data-translate="dedicated_server_interfaces_vrack_information_link_label"></a>
+</p>
+
+<div class="row">
+    <div class="col-md-8 mb-5"
+        data-ng-if="$ctrl.ola.isAvailable() && (!$ctrl.ola.isActivated() || !$ctrl.ola.isConfigured())">
+        <ola-step-checker class="h-100 step-checker"
```

Comment:
Shouldn't the class always be applied to the component ?

Reply:
It's a custom component. In fact, we could use directly the tagname for the selector here.